README.md

```diff
@@ -83,14 +86,20 @@ npm install -g shardus
 npm update @shardus/archiver
 ```
 
-## Running the Network
+## Running the Network Locally
 
 To start a local Shardeum network with 10 nodes, run:
 
 ```bash
 shardus start 10
 ```
 
+This command will start a Shardeum network of 10 nodes, 1 archive server and 1 monitor server exposed through port number 4000 and 3000 respectively. You can inspect the nodes via the `shardus pm2 list` command.
+
+> **Note:** If one of the services fails to start, it might be that the port is still in use from a previous session.
+
+For usage instructions and available options for the shardus command-line tool, run `shardus --help`
```

Comment:
this was removed in the original doc cleanup PR https://github.com/shardeum/shardeum/pull/109